.github/workflows/ci.yml

```diff
@@ -85,9 +90,13 @@ jobs:
         with:
           path: ${{ steps.yarn-cache.outputs.dir }}
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+      - name: Get arch value
+        run: echo "RCT_NEW_ARCH_ENABLED=${{ matrix.arch == 'NewArch' && '1' || '0' }}" >> $GITHUB_ENV
       - name: Install Dependencies
         run: yarn && yarn bootstrap
       - name: Install Podfiles
         run: cd example && npx pod-install
+      - name: Print React Native Info
+        run: cd example && npx react-native info
```

Comment:
just printing rn setup info, to make sure correctness of arch type

<img width="343" alt="Screenshot 2024-06-14 at 7 31 04 PM" src="https://github.com/react-native-menu/menu/assets/1898138/cf9792f4-225b-468c-9948-8b91620bc812">  <img width="325" alt="Screenshot 2024-06-14 at 7 30 37 PM" src="https://github.com/react-native-menu/menu/assets/1898138/0815e1f6-3b5e-4fe8-ab9c-1a56b7b6bd4d">